runtime/oti/j2sever.h

```diff
@@ -27,24 +27,17 @@
  * note: J2SE_15 needed for shared classes cache introspection but not supported by JVM.
  */
 /*
- * Note: J2SE_LATEST has to be updated to highest Java version supported by VM
- * 		 This allows JVM operates with latest version when classlib.properties doesn't present.
- * Additional note:
- * 		 J2SE_LATEST will be kept as J2SE_19 while Java 18.3 raw build is being built.
- * 		 When a separate Java 18.3 binary is required, a #ifdef flag will be introduced to define
- * 		 J2SE_LATEST to different values accordingly such as following:
- * 		 #if J9VM_JAVA9_BUILD > 9
- * 		 	#define J2SE_LATEST  J2SE_1803
- * 		 #else
- * 		 	#define J2SE_LATEST  J2SE_19
- * 		 #endif
+ * Note: J2SE_LATEST is the highest Java version supported by VM.
+ * 		 This allows JVM operates with latest version when neither classlib.properties
+ * 		 nor release file presents.
  */
 #define J2SE_15  0x1500
 #define J2SE_16  0x1600
 #define J2SE_17  0x1700
 #define J2SE_18  0x1800
 #define J2SE_19  0x1900
-#define J2SE_LATEST  J2SE_19
+#define J2SE_B7  0xB700				/* 0xB7 is 183 which refers to Java 18.3 */
```

Comment:
You can use B7 for the value, but the #define needs a better name, like J2SE_18_3

Reply:
`J2SE_18_3` was my initial choice, just felt it is easy to confuse with `J2SE_18`.

Reply:
We can brainstorm to find a better name. 
J2SE_2018_3 ?

Reply:
How about `JAVA_18_3` which differentiate from earlier name conversion `J2SE_xx`?

Reply:
Then its not or less obvious that JAVA_* and J2SE_* are related constants that are used interchangeably. Since 18.3 is short form for March 2018, I figure using 2018 would be clear and could not be confused with 18 meaning Java 8.

Reply:
We are using this definition to differentiate implementations in the code like "if (J2SE_VERSION((vm) >= J2SE_19)". J2SE_B7 would be larger then any of them (the check uses unsigned, does not it?) including 1.9.

Reply:
`J2SE_2018_3` works for me.

Reply:
@dmitripivkine right, that's the purpose, `Java 18.3 (0xB700)` should be superset of `Java 9 (0x1900)`.

Reply:
What we are going to do with Java 10 (0x1A00 I assume) ?

Reply:
`Java 18.3` is `Java 10` (the next version we thought). 
https://blogs.oracle.com/java-platform-group/faster-and-easier-use-and-redistribution-of-java-se